addon/PageScraper.js

```diff
@@ -33,17 +34,30 @@ PageScraper.prototype = {
    * HTML and insert it in the metadata database
    */
   _asyncParseAndSave: Task.async(function*(rawHTML, url) {
+    let event = this._tabTracker.generateEvent();
+    this._tabTracker.handlePerformanceEvent(event, "receivedRawHTML", Date.now());
```

Comment:
prepend this with metadata, so all the events are 'metadataThisThat' makes it easier to sort through them later